### PR TITLE
Fixes #203

### LINKED ISSUE #203: Confirmation in Deleting Uploaded Pics From Fragment
**Actual Behaviour**

When the user selects to delete photos from uploaded section, the photos just get deleted without any dialogue box for confirmation.

**Expected Behaviour**

There should be a confirmation Alert dialogue box in case the user accidentally presses the delete icon

**Would you like to work on the issue?**

Already Working on it. @Abhi2424shek  Should i send the PR ?

### PR DESCRIPTION
**Pull request by madhavrathi**
Fixes issue #203 

Changes: Added Alert Dialog for confirmation from end user

Screenshots for the change: 
screenshot_2017-03-20-15-27-53-391_vn mbm phimp me

### CODE REVIEW

**Comment by madhavrathi** [comment]
@Abhi2424shek  Please review

File: app/src/main/java/vn/mbm/phimp/me/Upload.java [hunk 1]
```diff
@@ -295,16 +299,39 @@ public void onClick(View v) {
...
-                    removableList.clear();
-                    ((ImageAdapter) listPhotoUpload.getAdapter()).notifyDataSetChanged();
-                    panelLable.setText(getResources().getString(R.string.upload));
-                    toggleButtonPanel(false);
-                    if (uploadGridList.isEmpty()) {
-                        noPhotos.setVisibility(View.VISIBLE);
-                    }
-                }
+                AlertDialog.Builder d = new AlertDialog.Builder(getContext());
+                d.setMessage("Are you sure ?").
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/Upload.java]: add hardcoded string in `string.xml`.

Comment (madhavrathi) [inline app/src/main/java/vn/mbm/phimp/me/Upload.java]: @pa1pal Fixed that.

**Comment by madhavrathi** [comment]
@Abhi2424shek Please Review

**Pull request merged**